jina/executors/indexers/__init__.py

```diff
@@ -190,6 +199,17 @@ class BaseKVIndexer(BaseIndexer):
     It can be used to tell whether an indexer is key-value indexer, via ``isinstance(a, BaseKVIndexer)``
     """
 
+    def add(self, docs: Iterator['jina_pb2.Document'], *args, **kwargs):
```

Comment:
I'd add a docstring, which hints, that the key will always be the `id` of the documents.